Source/SwiftLintFramework/Rules/RuleConfigurations/EmptyCountConfiguration.swift

```diff
@@ -0,0 +1,26 @@
+private enum ConfigurationKey: String {
+    case severity = "severity"
+    case onlyAfterDot = "only_after_dot"
+}
+
+public struct EmptyCountConfiguration: RuleConfiguration, Equatable {
+    private(set) var severityConfiguration = SeverityConfiguration(.warning)
```

Comment:
the existing default is `error`, can you change it to keep the same behavior?

Reply:
Updated.

Sorry, I didn't notice it was `.error before